fedora/__init__.py

```diff
@@ -373,3 +377,13 @@ async def whoowns(self, evt: MessageEvent, package: str) -> None:
         resp = "".join([x for x in [owners, admins, committers] if x != ""])
 
         await evt.reply(resp)
+
+    @command.new(help="Retrive a user's pronouns")
+    @command.argument("username", pass_raw=True, required=False)
+    async def pronouns(self, evt: MessageEvent, username: str) -> None:
+        # XXX: Why does this sometimes return a string?
+        user = self._get_fasuser(username, evt)
+        if isinstance(user, str):
```

Comment:
_get_user() returns a string if it hits an issue, and the string is what we have to respond to the user. there is probably a better way of doing this TBH

Reply:
Ah, okay. I'll just have it return the error message directly.